app/onepassword_events_api/appserver/static/javascript/components/wizard.js

```diff
@@ -0,0 +1,179 @@
+            e("div", { className: "token block" }, [
+              e("input", {
+                type: "text",
+                placeholder: "Enter your token here",
+                value: this.state.authToken,
+                onChange: (e) => {
+                  this.setState({
+                    authToken: e.target.value,
+                  });
+                },
+              }),
+            ]),
+          stepDetails.warning &&
+            (!this.state.success || this.state.error) &&
+            e("div", { className: "warning block" }, [
+              "Your other Splunk apps or add-ons may be able to access your Events API token. Make sure you trust them before you add your token.",
+            ]),
+          this.state.error &&
+            e("div", { className: "error block" }, [this.state.error]),
+          this.state.success &&
+            e("div", { className: "success block" }, [
+              "Your token has been successfully updated. If this is ",
+              e(
+                "strong",
+                null,
+                "the first time you're setting up 1Password Events Reporting for Splunk, you'll have to enable the scripted inputs."
+              ),
+              e("br"),
+              e("br"),
+              "If 1Password Events Reporting for Splunk ",
+              e(
+                "strong",
+                null,
+                "has already been setup, you'll have to disable and re-enable the scripted inputs for the changes to take effect."
+              ),
+              e("br"),
+              e("br"),
+              "For more information, check out the ",
+              e(
+                "a",
+                {
+                  target: "_blank",
+                  href: "https://support.1password.com/events-reporting-splunk",
+                },
+                "support article here."
+              ),
+            ]),
+        ]),
+        e("div", { className: "controls block" }, [
+          e("a", { onClick: this.handleSkip }, [
+            this.state.currentStep === 1
+              ? "I already have my Events API token"
+              : "",
+          ]),
+          e("div", { className: "controls-buttons" }, [
+            this.state.currentStep !== 1 &&
+              e("button", { className: "prev btn", onClick: this.handlePrev }, [
+                "Previous",
+              ]),
+            this.state.currentStep !== this.state.totalSteps
+              ? e(
+                  "button",
+                  { className: "next btn", onClick: this.handleNext },
+                  ["Next"]
+                )
+              : !this.state.success
+              ? e(
+                  "button",
+                  { className: "next btn", onClick: this.handleSubmit },
+                  ["Add Token"]
+                )
+              : e(
+                  "a",
+                  {
+                    href: "/app/onepassword_events_api",
```

Comment:
I think we defined the app name somewhere, do you think we should import it and use it here, rather than hardcoding? I don't feel strongly either way.